specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-07-01/virtualMachineScaleSet.json

```diff
@@ -3489,6 +3489,79 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}/virtualmachines/{instanceId}/attachDetachDataDisks": {
+      "post": {
+        "tags": [
+          "VirtualMachineScaleSetVMs"
+        ],
+        "operationId": "VirtualMachineScaleSetVMs_AttachDetachDataDisks",
+        "description": "Attach and detach data disks to/from a virtual machine in a VM scale set.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "vmScaleSetName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the VM scale set."
+          },
+          {
+            "name": "instanceId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The instance ID of the virtual machine."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./computeRPCommon.json#/definitions/AttachDetachDataDisksRequest"
+            },
+            "description": "Parameters supplied to the attach and detach data disks operation on a Virtual Machine Scale Sets VM."
+          },
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "schema": {
```

Comment:
>schema [](http://example.com/codeflow?start=13&length=6)

202 must not have schema:
[https://github.com/Azure/azure-openapi-validator/blob/main/docs/lro-post-return.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/lro-post-return.md)

Reply:
Adhering to this . 
Currently based on team discussion with team: 
Finalized HTTP Verb: HTTP POST
Response will contain both 200 and 202.
In 200, response "azure-asyncoperation" header will be provided. This is similar to start operation on the VM https://learn.microsoft.com/en-us/rest/api/compute/virtual-machines/start?tabs=HTTP#virtualmachine_start_maximumset_gen